addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -51,9 +51,7 @@
                                              <div class="float-right">
                                                 <t t-if="record.rating_percentage_satisfaction.raw_value &gt; 0">
```

Comment:
Same issue here, this condition is not correct either. It should depend on the number of ratings not their average.